cg/meta/archive/ddn_dataflow.py

```diff
@@ -38,13 +42,18 @@ class ResponseFields(str, Enum):
     RETRIEVE_FILES = "files/retrieve"
 
 
-class TransferData(BaseModel):
+class DataFlowFileTransferData(FileTransferData):
     """Model for representing a singular object transfer."""
 
     _metadata = None
     destination: str
     source: str
 
+    @classmethod
+    def from_models(cls, file: File, sample: Sample):
```

Comment:
from_file_and_sample maybe is clearer given that there are many models within cg?